components/script/dom/htmlimageelement.rs

```diff
@@ -174,18 +182,7 @@ impl PreInvoke for ImageContext {}
 
 impl HTMLImageElement {
     /// Update the current image with a valid URL.
-    fn update_image_with_url(&self, img_url: ServoUrl, src: DOMString) {
-        {
-            let mut current_request = self.current_request.borrow_mut();
-            current_request.parsed_url = Some(img_url.clone());
-            current_request.source_url = Some(src);
-
-            LoadBlocker::terminate(&mut current_request.blocker);
-            let document = document_from_node(self);
-            current_request.blocker =
-                Some(LoadBlocker::new(&*document, LoadType::Image(img_url.clone())));
-        }
-
+    fn fetch_image(&self, img_url: &ServoUrl) {
```

Comment:
There is a comment a few lines below that reads:
```
// This is a background load because the load blocker already fulfills the
// purpose of delaying the document's load event.
```
Since we are indeed creating and terminating load blockers as part of the "image request" cycle, do we need to create one more blocker here?